plugin/pkg/admission/resourcequota/admission.go

```diff
@@ -33,7 +33,7 @@ func init() {
 		func(client clientset.Interface, config io.Reader) (admission.Interface, error) {
 			// NOTE: we do not provide informers to the registry because admission level decisions
 			// does not require us to open watches for all items tracked by quota.
-			registry := install.NewRegistry(client, nil)
+			registry := install.NewRegistry(nil, nil)
```

Comment:
I assume this is because these aren't called (Get / List on the registry) - @derekwaynecarr can you sign off on this change if you haven't already?